changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+- Support multilevel bridge prefixes.
```

Comment:
Should the depth ssl cert thing be mentioned as well? Otherwise looks good to me.